fecgov/fec-cms
# Fix acorn vulnerability

## Summary (required)

Resolves #3605

- Fix acorn vulnerability
- Run `npm audit fix`

I tested this with a manual deploy to `dev` and testing locally

## Steps to test
- check out this branch
- run `npm i`
- run `npm run build`
- test locally
